### ISSUE TITLE
[Image V3] Unable to add an image without providing alternative text

### ISSUE THREAD

**Issue by lochdown** [adobe/aem-core-wcm-components#2710]
## Bug Report

**Current Behavior**
The mandatory "Alternative text for accessibility" text field is still showing as mandatory even if the "Don't provide an alternative text" checkbox is checked as there is an error message if this field is not filled out. This is forcing users to add alternative text, rendering the "Don't provide an alternative text" checkbox as useless.

Steps to replicate:
1. Click the image component -> Click the configure (wrench icon) button
2. Check the "Don't provide an alternative text" checkbox
3. Click Done
4. Uncheck the "Don't provide an alternative text" checkbox
5. The "Error: Please fill out this field." message will appear

<img width="590" alt="ImageDefect" src="https://github.com/adobe/aem-core-wcm-components/assets/154283103/22db7de0-23e4-4c3f-aef8-43393c0aeebc">

**Expected behavior/code**
When the "Don't provide an alternative text" checkbox is checked, the "Alternative text for accessibility" text field should be optional, allowing the user to submit their image without Alternative text.

**Environment**
- AEM version: 2023.12.14697.20231215T125030Z
- Core Components version: 2.23.4
- Image version: v3

**Possible Solution**
Alter the image javascript to make the "Alternative text for accessibility" text field not required when the Don't provide an alternative text" checkbox is checked.
